volontulo/tests/views/test_offers.py

```diff
@@ -475,3 +475,165 @@ def test_for_correct_slug(self):
         self.assertIn('offer', response.context)
         self.assertIn('volunteers', response.context)
         self.assertEqual(len(response.context['volunteers']), 5)
+
+
+class TestOffersJoin(TestCase):
+    u"""Class responsible for testing offer's join page."""
```

Comment:
This shouldn't be different class - this is Test_Offer "almost" as file name. Maybe we should clean this up and name classes as file names?